api/desecapi/management/commands/sync-from-pdns.py

```diff
@@ -20,12 +20,13 @@ def handle(self, *args, **options):
                     raise CommandError('{} is not a known domain'.format(domain_name))
 
         for domain in domains:
+            print(domain.name, '...', end='', flush=True)
```

Comment:
use `self.stdout.write` like in the [django docs example ](https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/)